cdc/capture/http_util.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package capture
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/cdc/entry"
+	"github.com/pingcap/ticdc/cdc/kv"
+	"github.com/pingcap/ticdc/cdc/model"
+	"github.com/pingcap/ticdc/cdc/sink"
+	"github.com/pingcap/ticdc/pkg/config"
+	cerror "github.com/pingcap/ticdc/pkg/errors"
+	"github.com/pingcap/ticdc/pkg/filter"
+	"github.com/pingcap/ticdc/pkg/util"
+	"github.com/pingcap/ticdc/pkg/version"
+	"github.com/r3labs/diff"
+	"github.com/tikv/client-go/v2/oracle"
+)
+
+// TODO: Rename the file and move it to other place. Clean the duplicate codes in ticdc/cmd/util.go
+func verifyUpdateChangefeedParameters(ctx context.Context, oldInfo *model.ChangeFeedInfo, config model.ChangefeedConfig) (*model.ChangeFeedInfo, error) {
```

Comment:
Maybe we could bind that method to ChangefeedConfig instead of putting it in util? Because it's not really related to http, but rather to the ChangefeedConfig checks.